exercises/concept/fruit-picker/.docs/hints.md

```diff
@@ -4,34 +4,26 @@
 
 - Callback functions are functions passed as an argument to a calling function.
 - Callback functions must meet the specification of the calling function.
-- These callbacks are all synchronous, meaning they all must return a value.
 
-## 1. Check if the grocer's service is online
+## 1. Try to notify with a successful message
 
-- Use the imported API function `checkStatus` in your function.
-- Pass a callback function to `checkStatus`. It should expect to receive a string argument.
-- Return the result from the `checkStatus` API function.
+- Use the imported API function `notify` in your function.
+- Pass an object to `notify`. It should expect to receive an object with a property of `message`. Think about what the message property should be.
+- Return the result from the `notify` API function.
```

Comment:
```suggestion
- Pay attention to the JSDoc type hint -- the function should not return a value.
```

The exemplar `onSuccess` and `onError` does not return a value from notify as they are marked as void returning functions